R/endpoints.R

```diff
@@ -1415,17 +1516,20 @@ paho_dengue <- function(regions,
   )
 }
 
-#' fetch Quidel data
-#'
-#' API docs: https://cmu-delphi.github.io/delphi-epidata/api/quidel.html
+#' Fetch Quidel data
```

Comment:
suggestion: "Fetch Quidel testing data" / "Fetch Quidel COVID-19 and influenza testing data"